authority/provisioner/scep.go

```diff
@@ -18,13 +18,20 @@ type SCEP struct {
 	ForceCN           bool     `json:"forceCN,omitempty"`
 	ChallengePassword string   `json:"challenge,omitempty"`
 	Capabilities      []string `json:"capabilities,omitempty"`
+	// IncludeRoots makes the provisioner return the CA root(s) in the GetCACerts response
+	IncludeRoots bool `json:"includeRoots,omitempty"`
```

Comment:
I believe this should be called `IncludeRoot`, the ca.json supports multiple roots for rotation purposes, but only one is really in use.